src/modules/user/openOrders/sagas/openOrdersCancelSaga.ts

```diff
@@ -3,11 +3,11 @@ import { call, put } from 'redux-saga/effects';
 import { API, RequestOptions } from '../../../../api';
 import { alertPush } from '../../../index';
 import { openOrdersCancelError, OpenOrdersCancelFetch } from '../actions';
-import { getCsrfToken } from '../../../../helpers';
+import { getCsrfToken, getOrderAPI } from '../../../../helpers';
 
 const ordersCancelOptions = (csrfToken?: string): RequestOptions => {
     return {
-        apiVersion: 'peatio',
+        apiVersion: getOrderAPI(),
```

Comment:
finex does not have this feature right now, peatio cannot do it instead of finex now too. I think it will be replaced with bulk cancel